src/common/metrics/location_summary.ts

```diff
@@ -3,6 +3,7 @@ import { Level, LevelInfoMap } from 'common/level';
 
 const LOW_NAME = 'Low';
 const MEDIUM_NAME = 'Moderate';
+const MEDIUM_HIGH_NAME = 'Moderate High';
```

Comment:
Hrm...  It took me a while to figure out where these get used (in SocialLocationPreview)...  Can we add a comment mentioning that?

Also, have you confirmed this is the copy we want?  It seems a bit inconsistent with the rest of the copy (low/medium/high/critical)

Reply:
Good catch! I also missed the MEDIUM_HIGH label on the `SocialLocationPreview` 🤦‍♂️  - I will add a note.

Reply:
I updated them to be consistent. I changed that based on the screenshot on the [copy document](https://docs.google.com/document/d/1Jsj4zOksCJcgjQLUo3rwVAobEUkGLFukD1K71IqVNpo/edit#) before the 4th color updates, but forgot about it after the updates

![image](https://user-images.githubusercontent.com/114084/83468635-3b64c000-a432-11ea-9819-80aeb3c5467c.png)

I flagged this to @maxwhenderson in the document to double-check what we want.